app/js/component/sad-question-card-footer/sad-question-card-footer.component.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+(() => {
+    var app = angular.module('sadApp');
+
+
+    function QuestionFooterController() {
+        var questionFooterCtrl = this;
```

Comment:
Por que usar o nome do questionário como nome da variável que guarda o contexto? O padrão self = this é mais conhecido, e compacto hauahauha